InteractiveHtmlBom/web/util.js

```diff
@@ -206,8 +206,10 @@ function initUtils() {
   if (config.fields.includes("Value")) {
     var index = config.fields.indexOf("Value");
     pcbdata.bom["parsedValues"] = {};
+    var bomList = getSelectedBomList().flat();
```

Comment:
This should always use `pcbdata.bom.both` otherwise some components will not be found if selected bom mode is F or B.